modules/installation-configuration-parameters.adoc

```diff
@@ -323,6 +323,10 @@ ifdef::osp[]
 |Additional security groups that are associated with compute machines.
 |A list of one or more UUIDs as strings. For example, `7ee219f3-d2e9-48a1-96c2-e7429f1b0da7`.
 
+|`compute.platform.openstack.zones`
+|{rh-openstack} Compute (Nova) availability zones (AZs) to install machines on. If this parameter is not set, the installer uses the default Nova zone or, if it is set, the value of `platform.openstack.zones`.
```

Comment:
We should also remove the reference to `platform.openstack.zones` here.

Reply:
@mandre So: 

> If this parameter is not set, the installer uses the default Nova zone.

?

Reply:
> If this parameter is not set, the installer uses the default Nova zone.

Exactly. `platform-openstack.zones` was my oversight sorry